build.sbt

```diff
@@ -62,6 +62,7 @@ lazy val generic = genericModule(project) in file("modules/generic") dependsOn `
 lazy val `generic-base` = genericModule(project) in file("modules/generic-base")
 lazy val hadoop = module(project) in file("modules/hadoop")
 lazy val http4s = module(project) in file("modules/http4s") dependsOn ip4s
+lazy val http4s022 = module(project) in file("modules/http4s022")
```

Comment:
removed dependency on `ip4s` since it is not necessary but brings a newer [ip4s](https://github.com/Comcast/ip4s) which is not compatible with **http4s v0.22.x**.

Reply:
If it's not necessary, can you please remove it from `http4s` as well to keep both modules in sync? This can also make it easier for users to switch between `http4s` and `http4s022` without hitting any unexpected errors (e.g. someone relying on `ip4s` through `http4s` that wants to move to `http4s022`).

Reply:
@ruippeixotog done, check it out please. Also just to doublecheck with you: I think this can be a potentially breaking change because there can be projects around that may have `ip4s` as a transient dependency via `http4s` module. But if you think it is not a big deal, then I'm fine with it too. Personally, I think it is always better to declare required dependencies explicitly after all.